File: rwfood.domain/Entities/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace rwfood.domain.Entities
{
    public class Product: BaseEntity
    {
        public int IdCategory { get; set; }
        public int Status { get; set; }
        public int Stock { get; set; }
        public string Code1 { get; set; }
        public string Code2 { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string? Image1 { get; set; }
        public string? Image2 { get; set; }
        public decimal Amount1 { get; set; }
        public decimal Amount2 { get; set; }
        public decimal Amount3 { get; set; }
        public decimal Amount4 { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: rwfood/src/interfaces/product.ts
export interface IProduct {
    id: number
    idCompany: number
    idCategory: number
    status: number
    createdAt: string
    name: string
    code1: string
    code2: string
    description: string
    image1: string
    image2: string
    amount1: number
    amount2: number
    amount3: number
    amount4: number
}

export interface IProductCategory {
    id: number
    idCompany: number
    idCategory: number
    status: number
    createdAt: string
    name: string
    code1: string
    code2: string
    description: string
    image1: string
    image2: string
    amount1: number
    amount2: number
    amount3: number
    amount4: number
    category: string
}
